### DISCUSSION TITLE
Display size (panels)

### DISCUSSION THREAD

**Discussion by bios4** [devemux86/cruiser#321] [Q&A] [answered]
Current version 3.0.32

Is there a reason behind the fact that with the setting "Settings > Application > Appearance > Display size (panels)" only the size of the LEFT panels is changed?

See examples:
- First image = panel size 95% (to match the size of the panels on the right)
- Second image = panel size 150% (panels on the right are not affected)

![Screenshot_20230921_114919_Cruiser](https://github.com/devemux86/cruiser/assets/145460147/13268e08-9d8f-43df-8623-99fb7323ce39)
![Screenshot_20230921_114941_Cruiser](https://github.com/devemux86/cruiser/assets/145460147/04ab8d42-4390-4ac5-82b0-4e3da9adf29a)

The only way to increase the panels on the right is to increase "Font size (panels)", which is set to 100% for both above screenshots.

**Comment by devemux86** [comment, marked as answer]
Indeed the "Display size (panels)" only affects the top-left navigation panel, which has as main component the direction icon.
(and thus also changes only the width of its bottom panels)

All small panels are mainly affected by the "Font size (panels)", as their main component is the text.

(probably we need better names in these settings)

**Comment by bios4** [comment]
> (probably we need better names in these settings)

Either that (suggestion: "Display size (navigation icon)").
Or change the implementation to apply the panel size setting to the right panels, also... ;-)

**Reply by devemux86** [reply]
> suggestion: "Display size (navigation icon)"

:+1:

**Reply by devemux86** [reply]
@bios4 thanks also for looking through Cruiser's many configuration settings!

If you find something that needs a better explanation, please let me know. :slightly_smiling_face:

**Comment by Ciutnik** [comment]
I agree that the name of the option is very misleading. Translating into Polish, I suggested something like "Instruction panel size" - there is no doubt what this value means 🙂

**Reply by airald** [reply]
It happened the same when translating to Catalan. It was a good chance to learn more about Cruiser.
